src/coreos-assembler

```diff
@@ -73,7 +73,13 @@ if [ -z "${cmd}" ]; then
 fi
 shift
 
-export COSA_META_SCHEMA="/usr/lib/coreos-assembler/schema/v1.json"
+COSA_META_SCHEMA="${COSA_META_SCHEMA:-/usr/lib/coreos-assembler/schema/v1.json}"
+schema_override="${PWD}/src/config/schema.json"
```

Comment:
I'm not sure we really need to support schema overrides like this at all; I think @jlebon suggested a generic `extensions` section that could contain anything.  So e.g. we'd eventually move the FCOS keys to:

```
extensions: {
   'fedora-coreos': {
     ...
  }
```
or so?

Reply:
I'm not *opposed* to it either to be clear, but AFAICS we don't have a use case for it now either.

Reply:
Up to you I guess...if you want to keep this code, OK by me and feel free to merge!

Reply:
Yeah, I lean towards getting this in for now and fleshing out the extension mechanism in a follow-up.